plugins/OnlineSales.Plugin.Sms/Services/GetshoutoutService.cs

```diff
@@ -1,59 +1,59 @@
-﻿// <copyright file="GetshoutoutService.cs" company="WavePoint Co. Ltd.">
```

Comment:
Hi @WASandaruwan, what happened to that file? It looks like it was completely rewritten, was not it?